### ISSUE TITLE
BigVGAN perfomance compared to Avocodo

### ISSUE THREAD

**Issue by bharaniyv** [DigitalPhonetics/IMS-Toucan#132]
Hi,
I saw that you are working on BigVGAN based vocoder in one of the Experimental branches and wanted to know how are performance improvements compared Avocodo and HiFiGAN vocoders especially is there any quality improvement for Unseen speakers in TTS?

**Comment by Flux9665** [comment]
In my experiments so far, the quality of the speech is improved, there are basically no artifacts. The speed on GPU is good, but on CPU it is extremely slow, compared to HiFiGAN. For unseen speakers, I did not really notice a difference, aside from the overall quality improvement. Both HiFiGAN and BigVGAN can handle unseen speakers wihtout problems. I will release a 24kHz BigVGAN model trained with Avocodo and HiFiGAN discriminators soon.

**Comment by bharaniyv** [comment]
That is great to hear and looking forward for the model release

**Comment by bharaniyv** [comment]
I just checked the performance of BigVGAN new release it is performing very well with my existing Indian languages model with very few artifacts even better than the Avocodo finetuned model. I finetuned Avocodo 48khz and 24khz models on Indian languages for 100k steps, 48khz model is performing better after finetuning than the 24khz version. 24khz is starting to deteriorate after 75k steps and sounds little distorted not sure what the reason is. I don't think there is any problem with the code since it is working perfectly with the 48khz version. Now I am thinking of finetuning Avocodo 48khz version on mel output of FastSpeech2 to reduce metallic noise. I think BigVGAN can sound even better with some finetuning, is it possible to release BigVGAN discriminator? it can help speedup the finetuning process alot.

Thanks
